start-building/tutorials/example-dapps/buy-me-a-coffee.md

```diff
@@ -0,0 +1,350 @@
+# Buy me a Coffee
+
+## Buy me a Coffee with dApp-Kit
+
+The outcome of this tutorial is a React application capable of identifying a user's wallet, sending tokens, and verifying the transaction on the Vechain network.
+
+You can open the resulting project on GitHub to check on all steps in parallel while reading each section.
+
+Here is the sequence we are building during this Tutorial:
+
+```mermaid
+sequenceDiagram
+participant User
+participant Wallet
+participant App
+participant Blockchain
+participant GitHub
+
+note over User, GitHub: Starting the App
+User->>App: open app
+App->>GitHub: get list of tokens
+GitHub-->>App: list of tokens
+App-->>User: show tokens and form to enter amount
+
+note over User, App: User Login
+User->>App: log in
+App->>Wallet: ask for user verification
+Wallet->>User: verify identity
+User-->>Wallet: approve verification
+Wallet-->>App: verification approved
+App-->>User: show user's wallet address
+
+note over User, Blockchain: Token Transfer
+User->>App: initiate token transfer
+App->>Wallet: request transaction signature
+Wallet->>User: request signature
+User-->>Wallet: provide signature
+Wallet-->>Blockchain: send transaction
+Blockchain-->>Wallet: transaction ID or error message
+Wallet-->>User: show confirmation
+Wallet-->>App: send transaction ID or error message
+
+loop until transaction is confirmed
+  App->>Blockchain: check transaction status
+  Blockchain-->>App: pending, successful, or failed
+end
+
+App-->>User: show transaction result
+```
+
+## Preparation
+
+This tutorial is based on a pre-existing React project that has already been configured with widely-used libraries. It will focus on incorporating Vechain-specific modules into this project. Therefore, only information related to Vechain will be covered in this article. It assumes a basic understanding of React, such as managing states and passing props, which will not be explained.
```

Comment:
"Vechain" should be "VeChain" in all the files, we are changing it everywhere

Reply:
The wording has been changed on all pages.